lib/rules/keyframes-name-pattern/index.js

```diff
@@ -38,6 +39,10 @@ const rule = (primary) => {
 				return;
 			}
 
+			if (!isStandardSyntaxKeyframesName(value)) {
```

Comment:
Let's move this to before the first conditionally as we normally check that things are standard before any other operation.